### PR TITLE
Items.py: added descriptions for all protoss units

### PR DESCRIPTION
**Pull request by ArchonofFail**
## What is this fixing or adding?
Added descriptions for all of the Protoss units from both WoL and LotV mostly based on the in-game descriptions with a few minor edits. Also updated the formatting for all units updated to match the formatting of the completed Terran units to make things look nicer and easily track what has been done.

## How was this tested?
It hasn't since I don't know how to but it's just text changes and I followed the instructions from Phanerus and matched the formatting of the completed descriptions so I don't think there should be any issues.

### CODE REVIEW

File: worlds/sc2/Items.py [hunk 1]
```diff
@@ -1250,23 +1251,54 @@ def get_full_item_list():
...
+                 classification=ItemClassification.filler,
+                 description="Increases the starting minerals for all missions by 15."),
+    ItemNames.STARTING_VESPENE: ItemData(801 + SC2WOL_ITEM_ID_OFFSET, "Vespene", 15, SC2Race.ANY, quantity=0, 
+                 classification=ItemClassification.filler,
+                 description="Increases the starting vespene for all missions by 15."),
     # This Filler item isn't placed by the generator yet unless plando'd
-    ItemNames.STARTING_SUPPLY: ItemData(802 + SC2WOL_ITEM_ID_OFFSET, "Supply", 2, SC2Race.ANY, quantity=0, classification=ItemClassification.filler),
+    ItemNames.STARTING_SUPPLY: ItemData(802 + SC2WOL_ITEM_ID_OFFSET, "Supply", 2, SC2Race.ANY, quantity=0,
+                 classification=ItemClassification.filler,
+                 description="Increases the starting supply for all missions by 2."),
```
Comment (MatthewMarinets) [inline worlds/sc2/Items.py]: The descriptions for starting resources / supply are incorrect; the amount of increase is configurable in the player's yaml. Defaults have been updated to 25 for gas / minerals and 5 for supply. Maybe just "Increases the starting supply on all missions."?

File: worlds/sc2/Items.py [hunk 2]
```diff
@@ -1250,23 +1251,54 @@ def get_full_item_list():
...
     # This item is used to "remove" location from the game. Never placed unless plando'd
-    ItemNames.NOTHING: ItemData(803 + SC2WOL_ITEM_ID_OFFSET, "Nothing Group", 2, SC2Race.ANY, quantity=0, classification=ItemClassification.trap),
+    ItemNames.NOTHING: ItemData(803 + SC2WOL_ITEM_ID_OFFSET, "Nothing Group", 2, SC2Race.ANY, quantity=0,
+                 classification=ItemClassification.trap,
+                 description="Does nothing. Used to 'remove' location from the game."),
```
Comment (MatthewMarinets) [inline worlds/sc2/Items.py]: Missing article, and the single quotes probably aren't necessary. "Used to remove a location..." or "Used to remove locations..."

File: worlds/sc2/Items.py [hunk 3]
```diff
@@ -1465,36 +1497,96 @@ def get_full_item_list():
...
+                 description="Ranged attack strider. Can use the Phase Blink and Phasing Armor abilities."),
+    ItemNames.SENTRY: ItemData(6 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 15, SC2Race.PROTOSS, 
+                 classification=ItemClassification.progression, origin={"lotv"},
+                 description="Robotic support unit can use the Guardian Shield ability and restore the shields of nearby Protoss units."),
+    ItemNames.ENERGIZER: ItemData(7 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 16, SC2Race.PROTOSS, 
+                 classification=ItemClassification.progression, origin={"lotv"},
+                 description="Robotic support unit. Can use the Chrono Beam ability and become stationary to power nearby structures."),
+    ItemNames.HAVOC: ItemData(8 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 17, SC2Race.PROTOSS,
+                 origin={"lotv"}, important_for_filtering=True,
+                 description="Robotic support unit. Can use the Target lock and Force Field abilities and increase the range of nearby Protoss units."),
```
Comment (MatthewMarinets) [inline worlds/sc2/Items.py]: Inconsistent capitalization on "Target lock"

File: worlds/sc2/Items.py [hunk 4]
```diff
@@ -1465,36 +1497,96 @@ def get_full_item_list():
...
+    ItemNames.SIGNIFIER: ItemData(9 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 18, SC2Race.PROTOSS, 
+                 classification=ItemClassification.progression, origin={"ext"},
+                 description="Potent cloaked psionic master. Can use the Feedback and Crippling Psionic Storm abilities. Can merge into an Archon."),
```
Comment (MatthewMarinets) [inline worlds/sc2/Items.py]: Maybe mention that they're permanently cloaked?

File: worlds/sc2/Items.py (hunk 1 above)
Comment (ArchonofFail) [reply worlds/sc2/Items.py]: oh right that makes sense, lemme fix rq

File: worlds/sc2/Items.py (hunk 3 above)
Comment (ArchonofFail) [reply worlds/sc2/Items.py]: fixed

File: worlds/sc2/Items.py (hunk 4 above)
Comment (ArchonofFail) [reply worlds/sc2/Items.py]: fixed

Comment (MatthewMarinets) [reply worlds/sc2/Items.py]: Whoops, my eyes skipped right over "cloaked" the first time around. Permanently cloaked still works, though so all good.

**Review by MatthewMarinets — APPROVED** [review]

**Pull request merged**